addons/auth_passkeys/models/passkey.py

```diff
@@ -0,0 +1,32 @@
+from odoo import fields, models, _
+
+class PassKey(models.Model):
+    _name = "auth.passkeys.key"
+    _description = "Passkeys"
+    _order = "id desc"
+
+    name = fields.Char(required=True)
+    credential_identifier = fields.Char(required=True)
+    public_key = fields.Char(required=True)
+    sign_count = fields.Integer(default=0)
+
+    def _get_nopw_user_by_id(self, id):
+        # We strip the = because webauthn base64 padding behaves a little different depending on the platform (win/yubi)
+        results = self.sudo().search([("credential_identifier", "like", id.strip("=")+"%")])
```

Comment:
Regarding that `strip("=")+"%"` and that comment 
> We strip the = because webauthn base64 padding behaves a little different depending on the platform (win/yubi)

Is that critical ?
Because to me that strip could be an attack vector, what an attacker could use to try bypassing the security.

If the goal of this strip is to be able to use the same yubikey/identifier when you move your yubikey from your linux to your windows laptop, then I think I would drop it and for the user to have two different passkeys, one for win and one for linux, instead of doing this trick.
This can be discussed, I just my first sight opinion.